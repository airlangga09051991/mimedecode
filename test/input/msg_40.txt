From test Sat Feb  1 00:00:00 2014
MIME-Version: 1.0
Content-Type: text/html; boundary="--961284236552522269"

----961284236552522269
Content-Type: text/html;
Content-Transfer-Encoding: 7Bit

<html></html>

----961284236552522269--
