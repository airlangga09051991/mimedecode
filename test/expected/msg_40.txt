From test Sat Feb  1 00:00:00 2014
MIME-Version: 1.0
Content-Type: text/plain; boundary="--961284236552522269"
X-MIME-Autoconverted: from text/html to text/plain by test id ./html2txt.py

----961284236552522269
Content-Type: text/html;
Content-Transfer-Encoding: 7Bit



----961284236552522269--

